target/linux/octeon/base-files/lib/preinit/01_sysinfo

```diff
@@ -17,6 +17,11 @@ do_sysinfo_octeon() {
 		name="erpro"
 		;;
 
+	"UBNT_E300"*)
+		# let 02_sysinfo handle it since it has its own device tree
```

Comment:
Please make it more clear that you are referring to _generic_ 02_sysinfo here.

Reply:
sure, adjusted commentary.